src/tiles/hover/tile-hover-pane.ts

```diff
@@ -23,25 +34,69 @@ export class TileHoverPane extends LitElement {
   protected render(): TemplateResult {
     return html`
       <div id="container">
-        <tile-list
-          .model=${this.model}
-          .baseNavigationUrl=${this.baseNavigationUrl}
-          .baseImageUrl=${this.baseImageUrl}
-          .loggedIn=${this.loggedIn}
-          .sortParam=${this.sortParam}
-          .collectionTitles=${this.collectionTitles}
-        ></tile-list>
+        ${this.headerTemplate}
+        <div id="hover-tile-list">
+          <tile-list
+            .model=${this.model}
+            .baseNavigationUrl=${this.baseNavigationUrl}
+            .baseImageUrl=${this.baseImageUrl}
+            .loggedIn=${this.loggedIn}
+            .sortParam=${this.sortParam}
+            .collectionTitles=${this.collectionTitles}
+            .mobileBreakpoint=${this.mobileBreakpoint}
+            .currentWidth=${this.currentWidth}
+          ></tile-list>
+        </div>
+      </div>
+    `;
+  }
+
+  private get headerTemplate(): TemplateResult | typeof nothing {
+    // early return if item does't have parent collection
+    if (this.model?.collections.length === 0) return nothing;
+
+    let collectionTitle = '';
+    let collectionIdentifier = '';
+
+    for (const collection of this.model?.collections || []) {
+      if (
+        !suppressedCollections[collection] &&
+        !collection.startsWith('fav-')
+      ) {
+        collectionTitle = this.collectionTitles?.get(collection) ?? collection;
+        collectionIdentifier = collection;
+        break;
+      }
+    }
+
+    // sometimes item does have collections but they are in suppressed or favorite list,
+    // let's not render that
+    if (!collectionIdentifier) return nothing;
+
+    return html`
+      <div id="list-line-header">
+        <a href="${this.baseNavigationUrl}/details/${collectionIdentifier}">
+          <img
+            src="${this.baseImageUrl}/services/img/${collectionIdentifier}"
+            alt=""
+          /><span>${collectionTitle}</span>
+        </a>
       </div>
     `;
   }
 
   static get styles(): CSSResultGroup {
+    const hoverPaneHeaderBGColor = css`var(--hoverPaneHeaderBGColor, #edf0ff)`;
+    const iaLinkColor = css`var(--ia-theme-link-color, #4b64ff)`;
+    const iaFontFamily = css`var(--ia-theme-base-font-family, "Helvetica Neue", Helvetica, Arial, sans-serif);`;
```

Comment:
q: wonder if we can just receive this implied, inherited from document?

Reply:
We are receiving these from document by just created variable here to group them at single place having fallback to insure they get correct values.